app-accessibility/flite/flite-2.1.recipe

```diff
@@ -0,0 +1,115 @@
+SUMMARY="A light-weight text-to-speech engine"
+DESCRIPTION="Flite is a text-to-speech engine developed by the Carnegie \
+Mellon University. As a light-weight alternative to the Festival engine, it \
+is desinged to fit into small embedded machines as well as large-scale \
+servers. In addition to its native format flite supports voices generated by \
+the FestVox project as well."
+HOMEPAGE="http://www.festvox.org/flite/"
+COPYRIGHT="1986 University of Toronto
+	1992, 1995 Markus Mummert
+	1996-2004 University of Edinburgh, UK
+	1999-2014 Language Technologies Institute, Carnegie Mellon University
+	2001 Cepstral, LLC
+	2001 Sun Microsystems, Inc.
+	2003-2004 Nagoya Institute of Technology
+	2003-2004 Nara Institute of Science and Technology
+	2008-2012 University of Illinois at Urbana-Champaign"
+# Although individually many files have different licenses, as a whole this
+# license covers all the files.
+LICENSE="Flite"
+REVISION="3"
+SOURCE_URI="http://www.festvox.org/flite/packed/flite-2.1/flite-${portVersion}-release.tar.bz2"
+CHECKSUM_SHA256="c73c3f6a2ea764977d6eaf0a287722d1e2066b4697088c552e342c790f3d2b85"
+SOURCE_DIR="flite-${portVersion}-release"
+PATCHES="flite-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```

Comment:
Mark x86_gcc2 as broken (or maybe fix the parse errors)

Reply:
Strike that, the previous patch should have solved that https://github.com/haikuports/haikuports/blob/9157c823aee653229d9da28843252f116e7265fa/app-accessibility/flite/patches/flite-2.0.0.patchset#L53